<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RatBehaviorAnalysisClass</class>
 <widget class="QMainWindow" name="RatBehaviorAnalysisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RatBehaviorAnalysis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="depthLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>512</width>
      <height>424</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="lowerBound">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>4500</number>
       </property>
       <property name="value">
        <number>450</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="upperBound">
       <property name="minimum">
        <number>400</number>
       </property>
       <property name="maximum">
        <number>4500</number>
       </property>
       <property name="value">
        <number>1850</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lower(mm)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Upper(mm)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Depth Sensor Range&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Depth Display</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Grayscale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_PseudoColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pseudo Color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_TrialNo"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Trial Number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Recording Format</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="recordingFormat">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Video</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Raw Data</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="recordingFPS">
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="recordingButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Background Refer</string>
    </property>
    <widget class="QPushButton" name="backgroundButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_BGR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Ready</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backgroundSaveButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Depth ROI Selection</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>237</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Point 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="point2_x">
        <property name="maximum">
         <number>512</number>
        </property>
        <property name="value">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="point2_y">
        <property name="maximum">
         <number>424</number>
        </property>
        <property name="value">
         <number>280</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="point1_y">
        <property name="maximum">
         <number>424</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="point1_x">
        <property name="maximum">
         <number>512</number>
        </property>
        <property name="value">
         <number>130</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Point 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="ROI_Off">
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="ROI_On">
        <property name="text">
         <string>ON</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="calculationButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Feature Calculation</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="featureDisplay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threshold">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSwitch">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>With
Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnLog">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear
Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTracking">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh
Tracking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnExpand">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Expand&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Color ROI Selection</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>237</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="1">
       <widget class="QSpinBox" name="color_point2_x">
        <property name="maximum">
         <number>1920</number>
        </property>
        <property name="value">
         <number>1300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Point 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="color_point1_y">
        <property name="maximum">
         <number>1080</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="color_point1_x">
        <property name="maximum">
         <number>1920</number>
        </property>
        <property name="value">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="color_point2_y">
        <property name="maximum">
         <number>1080</number>
        </property>
        <property name="value">
         <number>800</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Point 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnSwitchTimeStampColor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch
Time Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ratbehavioranalysis.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>recordingButton</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>recordingButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundButton</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>backgroundButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>backgroundSaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculationButton</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>calculationButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnSwitch</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>BtnSwitchClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnTracking</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>BtnTrackingClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnLog</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>BtnLogClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnExpand</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>BtnExpandClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnSwitchTimeStampColor</sender>
   <signal>clicked()</signal>
   <receiver>RatBehaviorAnalysisClass</receiver>
   <slot>BtnSwitchTimeStampColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>recordingButtonClicked()</slot>
  <slot>backgroundButtonClicked()</slot>
  <slot>backgroundSaveButtonClicked()</slot>
  <slot>calculationButtonClicked()</slot>
  <slot>BtnSwitchClicked()</slot>
  <slot>BtnTrackingClicked()</slot>
  <slot>BtnLogClicked()</slot>
  <slot>BtnExpandClicked()</slot>
  <slot>BtnSwitchTimeStampColorClicked()</slot>
 </slots>
</ui>
